src/Reusables/Navbar.js

```diff
@@ -0,0 +1,33 @@
+
+function Navbar() {
+    return (
+        <header class="nav-header">
+        <div class="nav-header-div">
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">HOME</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">FILTER</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#top">PRODUCTS</a>
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">   
+                        <a href="#top" class="btn-small margin-lr">Login</a>
```

Comment:
since you are using react-router, you can use Link or NavLink Tag for the nav links

Reply:
I'm keeping it as it is for now, once will add the respective pages, will surely convert them to Link